assets/script.js

```diff
@@ -0,0 +1,203 @@
+// Declaring global variables
+var today = dayjs();
+var full_name = "";
+var index;
+var tarotDesc = [];
+var tarotName = [];
+var historySign;
+var historyName;
+var tarotID;
+
+today = today.format('MM/DD/YYYY');
+// Empty the current horoscope section of the page
+function clear() {
+    console.log("inside clear");
```

Comment:
Try to remove `console.log` statements before deploying to production.

There are multiple reasons for this

1. Console.logs negatively impact performance on the client side
2. You give end-users unnecessary insight in to your program
3. Less code is almost always better, having to maintain lots of console.logs is a pain

Reply:
This is some good insight Alex. I always saw console.log as a scaffolding or comforter to use in the code.